src/components/Book.js

```diff
@@ -1,36 +1,27 @@
-const Book = () => (
-  <>
-    <div className="card">
-      <div className="title">
-        <span>Action</span>
-        <h1>The hunger games</h1>
-        <p>Unseen Content</p>
-        <ul>
-          <li>Comment</li>
-          <li>Remove</li>
-          <li>Edite</li>
-        </ul>
-      </div>
-
-      <div className="progress">
-        <h4>60%</h4>
-      </div>
-      <div className="chapter">
-        <h4>Current Chapter</h4>
-        <h5>Chapter 16</h5>
-        <button type="button">UPDATE PROGRESS</button>
-      </div>
-    </div>
+import { useSelector, useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/books';
 
-    <div className="card">
+const Book = () => {
+  const books = useSelector((state) => state.booksReducer);
+  const dispatch = useDispatch();
+  return (books.map((book) => (
+    <div key={book.id} className="card">
```

Comment:
- [ ] [OPTIONAL] Usually this code which gets all the books in the store and iterates through should be done in the `Books` component. Your singular `Book` component should simply render the individual books that were passed in as props to it from the `Books` component👍